BT_subtree.py

```diff
@@ -0,0 +1,20 @@
+class Node:
+    def __init__(self, value):
+        self.value = value 
+        self.left = None
+        self.right = None
+
+def  equal(t1, t2):
+    if t1 is None or t2 is None:

```

Comment:
Didn't quite understand this condition check - if only one of either t1 or t2 is none, then what happens in this code?